build/rpm/custom-strip

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "STRIPPING IN $*"
+
+skip=(
+  "${RPM_BUILD_ROOT}/usr/libexec/pcp/pmdas/bpf/modules/runqlat.bpf.o"
```

Comment:
let's put a wildcard there instead of hard coding the module names, otherwise it will bit us for future modules

Reply:
fixed in cb8b03215